### PR TITLE
Add version: OpenJS.NodeJS.LTS version 6.17.0

### PR DESCRIPTION
**Pull request by coolplaylinbot**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/144795)

### CODE REVIEW

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Please see https://www.github.com/microsoft/winget-pkgs/pull/144865#issuecomment-2002127589

File: manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2019-02-28
+AppsAndFeaturesEntries:
+- DisplayName: Node.js
+  UpgradeCode: '{47C07A3A-42EF-4213-A85D-8F5A59077C28}'
```
Comment (aisgbnok) [inline manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.installer.yaml:17]:
```suggestion
```

> Both `OpenJS.NodeJS` and `OpenJS.NodeJS.LTS` are using the same `UpgradeCode` and adding it to the former one will cause the latter to be recognized as the former. (And adding it to both may cause some users unable to update)


- https://github.com/microsoft/winget-pkgs/pull/157605#issuecomment-2164167842

File: manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  InstallerSha256: E3478FCF22E3395A41F34CF7E64527E6CC532C109E5A8D67AB1DE213E53F780C
+  ProductCode: '{512D50AB-E625-4D27-9AFF-2F2FFBAC2AC3}'
+- Architecture: x64
+  InstallerUrl: https://nodejs.org/dist/v6.17.0/node-v6.17.0-x64.msi
+  InstallerSha256: 3A9171AB366BA45ACCC2EC898B50390D0020797B1CF16A16E9F3E105FCFDC95A
+  ProductCode: '{99D1C612-99E8-4857-86FE-13010D3FF067}'
+- Architecture: arm64
+  InstallerUrl: https://nodejs.org/dist/v6.17.0/node-v6.17.0-x64.msi
+  InstallerSha256: 3A9171AB366BA45ACCC2EC898B50390D0020797B1CF16A16E9F3E105FCFDC95A
+  ProductCode: '{99D1C612-99E8-4857-86FE-13010D3FF067}'
```
Comment (aisgbnok) [inline manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.installer.yaml:31]:
```suggestion
Installers:
- Architecture: x86
  InstallerUrl: https://nodejs.org/dist/v6.17.0/node-v6.17.0-x86.msi
  InstallerSha256: E3478FCF22E3395A41F34CF7E64527E6CC532C109E5A8D67AB1DE213E53F780C
  ProductCode: '{512D50AB-E625-4D27-9AFF-2F2FFBAC2AC3}'
- Architecture: x64
  InstallerUrl: https://nodejs.org/dist/v6.17.0/node-v6.17.0-x64.msi
  InstallerSha256: 3A9171AB366BA45ACCC2EC898B50390D0020797B1CF16A16E9F3E105FCFDC95A
  ProductCode: '{99D1C612-99E8-4857-86FE-13010D3FF067}'
```

File: manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: OpenJS.NodeJS.LTS
+PackageVersion: 6.17.0
+InstallerLocale: en-US
```
Comment (aisgbnok) [inline manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.installer.yaml:6]:
```suggestion
```

File: manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+PackageIdentifier: OpenJS.NodeJS.LTS
+PackageVersion: 6.17.0
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
```
Comment (aisgbnok) [inline manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.installer.yaml:13]:
```suggestion
Scope: machine
InstallerSwitches:
  InstallLocation: INSTALLDIR="<INSTALLPATH>"
UpgradeBehavior: install
Commands:
- node
- npm
FileExtensions:
- js
```

File: manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Author: OpenJS Foundation
+PackageName: Node.js LTS
+PackageUrl: https://nodejs.org/
+License: MIT
+LicenseUrl: https://github.com/nodejs/node/blob/main/LICENSE
+ShortDescription: Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine.
+Description: |-
+  Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine.
+  Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.
+  Node.js" package ecosystem, npm, is the largest ecosystem of open source libraries in the world.
```
Comment (aisgbnok) [inline manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.locale.en-US.yaml:20]:
```suggestion
PublisherUrl: https://openjsf.org/
PublisherSupportUrl: https://github.com/nodejs/node/blob/main/.github/SUPPORT.md
PrivacyUrl: https://privacy-policy.openjsf.org/
Author: OpenJS Foundation
PackageName: Node.js (LTS)
PackageUrl: https://nodejs.org/
License: MIT
LicenseUrl: https://github.com/nodejs/node/blob/v6.17.0/LICENSE
Copyright: Copyright Node.js contributors. All rights reserved.
CopyrightUrl: https://trademark-policy.openjsf.org/
ShortDescription: Run JavaScript Everywhere
Description: Node.js® is a free, open-source, cross-platform JavaScript runtime environment that lets developers create servers, web apps, command line tools and scripts.
```

File: manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+Moniker: nodejs-lts
+Tags:
+- cross-platform
+- foss
+- javascript
+- lts
+- node
+- nodejs
+- npm
```
Comment (aisgbnok) [inline manifests/o/OpenJS/NodeJS/LTS/6.17.0/OpenJS.NodeJS.LTS.locale.en-US.yaml:29]:
```suggestion
- coding
- cross-platform
- develop
- development
- environment
- javascript
- js
- lts
- node
- npm
- programming
- runtime
- v8
ReleaseNotesUrl: https://github.com/nodejs/node/releases/tag/v6.17.0
Documentations:
- DocumentLabel: Learn
  DocumentUrl: https://nodejs.org/en/learn/
- DocumentLabel: Documentation
  DocumentUrl: https://nodejs.org/docs/v6.17.0/api/
- DocumentLabel: About
  DocumentUrl: https://nodejs.org/about/
```

**Review by aisgbnok — CHANGES_REQUESTED** [review]
To note, I don't see the appeal to adding versions of [NodeJS 6 as it has been EOL for 5+ years.](https://nodejs.org/en/about/previous-releases#:~:text=v6.17.1,2019%2D04%2D03) If anything, I would add `6.17.1` as it is the last release of NodeJS 6.

But if you want to:

---

CC @CoolPlayLin

**Review by aisgbnok — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Comment by mdanish-kh** [comment]
[Policy] Needs-Manual-Merge

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**